utils/frame/remote-externalities/src/lib.rs

```diff
@@ -375,49 +376,62 @@ impl Builder {
 	}
 }
 
-#[cfg(feature = "remote-test")]
 #[cfg(test)]
-mod tests {
-	use super::*;
+mod test_prelude {
+	pub use super::*;
+	pub use sp_runtime::testing::{H256 as Hash, Block as RawBlock, ExtrinsicWrapper};
 
-	fn init_logger() {
+	pub type Block = RawBlock<ExtrinsicWrapper<Hash>>;
+
+	pub fn init_logger() {
 		let _ = env_logger::Builder::from_default_env()
 			.format_module_path(false)
 			.format_level(true)
 			.try_init();
 	}
+}
 
-	#[async_std::test]
-	async fn can_build_one_pallet() {
+#[cfg(test)]
+mod tests {
+	use super::test_prelude::*;
+
+	#[tokio::test]
+	async fn can_load_cache() {
 		init_logger();
-		Builder::new()
-			.mode(Mode::Online(OnlineConfig {
-				modules: vec!["Proxy".into()],
-				..Default::default()
+		Builder::<Block>::new()
+			.mode(Mode::Offline(OfflineConfig {
+				cache: CacheConfig { name: "test_data/proxy_test".into(), ..Default::default() },
 			}))
 			.build()
 			.await
 			.unwrap()
 			.execute_with(|| {});
 	}
+}
 
-	#[async_std::test]
-	async fn can_load_cache() {
+#[cfg(feature = "remote-test")]
+#[cfg(test)]
+mod remote_tests {
+	use super::test_prelude::*;
+
+	#[tokio::test]
+	async fn can_build_one_pallet() {
 		init_logger();
-		Builder::new()
-			.mode(Mode::Offline(OfflineConfig {
-				cache: CacheConfig { name: "proxy_test".into(), ..Default::default() },
+		Builder::<Block>::new()
+			.mode(Mode::Online(OnlineConfig {
+				modules: vec!["Proxy".into()],
+				..Default::default()
 			}))
 			.build()
 			.await
 			.unwrap()
 			.execute_with(|| {});
```

Comment:
My original tests were not really sophisticated, and I think you can do better here: 

We need a way to assert that _some_ data has _actually_ been injected into to the proxy pallet. For example, if you scrape the system pallet, now you want to try and read something like `System::block_number` or some other storage item that is unlikely not to change. 

Or, you can assume that you know all storage items in proxy are prefixed with `twox128("Proxy")`, and then assert that `sp_io::storage::next_key(twox128("Proxy"))` is some. 

There are other ways to also do this. 

Treat it as a bonus, if you want to explore the storage a bit.